lib/rules/prefer-screen-queries.ts

```diff
@@ -110,3 +119,8 @@ export default ESLintUtils.RuleCreator(getDocsUrl)<Options, MessageIds>({
     };
   },
 });
+
+function usesContainerOrBaseElement(node: TSESTree.CallExpression) {
+  const secondArgument = node.arguments[1]
+  return isObjectExpression(secondArgument) && secondArgument.properties.some((property) => isProperty(property) && isIdentifier(property.key) && ALLOWED_RENDER_PROPERTIES_FOR_DESTRUCTURING.includes(property.key.name))
+}
```

Comment:
Point of discussion: This should not be declared in top of the file? I don't know if we're following some pattern, but generally I prefer to put all utilities before default export

Reply:
I thought we had a linter rule for this.

Reply:
We don't have something to for this as far as I remember. In some rules is at the top, in others at the bottom.

Reply:
> We don't have something to for this as far as I remember. In some rules is at the top, in others at the bottom.

Can we add this rule? or is not necessary in this project?

Reply:
I'm not sure which ESLint rule cover this, but I guess it's better to address it on a separate ticket as it will implies fixing all reported issues (which is not directly related to this rule).

Reply:
Hmmm I really thought it was included in the recommended config, and that I moved all the utilities to the top because I had linting errors 🤔

Reply:
just in case I moved above 😅